Testreglar/1.3.2/App/app-1.3.2a.json

```diff
@@ -0,0 +1,157 @@
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva appside testar du?",
+			"ht": "<p>Beskriv appsida med få stikkord, eller vel i lista.</p>",
+			"type": "tekst",
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har appsida innhald, der meininga blir påvirka av rekkefølgen innhaldet blir presentert i?",
+			"ht": "<ul><li>Ei rekkefølge er meiningsfull dersom rekkefølgen på innhaldet ikkje kan endrast utan å endre meininga til innhaldet.<ul><li>Eksempel på innhald med i ei meiningsfylt rekkefølgje er tekst, matematiske formlar, tabellar og nummererte lister.</li></ul></li></ul><p><strong>Merk:</strong> Unummererte lister har ikkje ei meiningsfylt rekkefølgje.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appside har ikkje innhald der meininga blir påvirka av rekkefølgen det blir presentert i."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Er det mogleg å sveipe til alt innhald på appsida?",
+			"ht": "<p>Viss det er ikkje mogleg å sveipe til alt innhold, kan du ikkkje testa leserekefølgen for innhaldet på appsida.</p><p>Slik testar du:</p><ul><li>Aktiver skjermlesar (opplesningsfunksjon)<ul><li>iOS: VoiceOver</li><li>Android: Talkback</li></ul></li><li>Opne appsida du testar</li><li>Sjekk om det er mogleg å sveipe til alt innhald </li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ikkje testbart",
+					"utfall": "Det er ikkje mogleg å sveipe til alt innhald på appsida."
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Er visuell og programmatisk bestemt leserekkefølge på appsida lik?",
+			"ht": "<p>Vi brukar skjermlesar for å sjekke programmatisk bestemt leserekkefølge. </p><p>Slik testar du:</p><ul><li>Åpne appsida og få innhaldet lest opp med skjermlesar<ul><li>Sveip til høgre for å lese opp innhald</li><li>Sveip til venstre dersom du treng å gå tilbake i innhaldet</li></ul></li><li> Samanlikn visuell leserekkefølge på appsida, med opplesinga frå skjermlesaren.<ul><li>Les appsida frå toppen og nedover, frå venstre mot høgre.</li><li>Dette gjeld også der innhald er delt opp i blokker. Les blokkene frå toppen og ned, frå venstre mot høgre.</li><li>Det kan vera at du må opne menyar, modaler og liknande, for å teste alt innhald.</li></ul></li><li>Sjekk om innhaldet har same rekkefølgje visuelt og med bruk av skjermlesar</li></ul>",
```

Comment:
Er mye enklere å bare dra ned med to fingre og få alt innhold lest opp mens du følger med på fokusmarkeringen fra skjermleseren

Reply:
Gjelder det bare android?

Reply:
Lagt inn som et alternativ.